roles/setup-local/templates/acsf.aliases.drushrc.php

```diff
@@ -63,8 +63,15 @@ function ($var) use ($alias_key) {
 $stack_id = array(
   "cardinalsites" => "01",
   "cardinald7" => "02",
+  "leland" => "03",
 );
 
+// Set the string for sites.stanford.edu or people.stanford.edu.
+$sites_people = "sites";
```

Comment:
Oh, I took a slightly different approach. 

For flexibility I would assume less about the urls. Check out https://github.com/SU-SWS/acsf-cardinald7/pull/80

Reply:
Yeah, I think I'm going to poach what you did in https://github.com/SU-SWS/acsf-cardinald7/pull/80. Don't let the aliases file hold up this PR; I can do a separate one to catch up the aliases.